packages/app-page-builder/src/admin/views/PageBlocks/BlocksByCategoriesDataList.tsx

```diff
@@ -109,13 +114,28 @@ const BlocksByCategoriesDataList = ({ canCreate }: PageBuilderBlocksByCategories
         listQuery?.data?.pageBuilder?.listBlockCategories?.data || [];
     const pageBlocksData: PbPageBlock[] = listQuery?.data?.pageBuilder?.listPageBlocks?.data || [];
 
-    const filterData = useCallback(
-        ({ slug, name }) => {
-            return slug.toLowerCase().includes(filter) || name.toLowerCase().includes(filter);
+    const filterBlocksData = useCallback(
+        ({ name }) => {
+            return name.toLowerCase().includes(filter);
         },
         [filter]
     );
 
+    const filteredBlocksData: PbPageBlock[] =
+        filter === "" ? pageBlocksData : pageBlocksData.filter(filterBlocksData);
+
```

Comment:
Looks good. Can we maybe move this data filtering / sorting code into a separate function or React hook?

Something like:

```ts
// packages/app-page-builder/src/admin/views/PageBlocks/hooks/useCategoriesListData.ts

type BlockListData = Array<...>; // Please add the correct type.
type BlockCategoriesListData = Array<...>; // Please add the correct type.

function useCategoriesListData(blockCategoriesData, blockData): [BlocksList, BlockCategoriesList];
```